kubernetes.go

```diff
@@ -268,38 +268,35 @@ func (c *KubernetesClient) Watch() error {
 		switch e.Type {
 		case watch.Added:
 			s.Update(e.Object)
-			updater.Signal()
 		case watch.Modified:
 			old := s.Update(e.Object)
 			if old == nil {
 				log.Println("Modified unknown object, this shouldn't happen")
```

Comment:
Should we do a "return" here to avoid modifying unknown objects ? I guess in this case it would be similar as create a new object